packages/components/PrefixBadgePicker/PrefixBadgePicker.module.css

```diff
@@ -0,0 +1,137 @@
+.PrefixBadgePicker {
+  display: flex;
+  align-items: center;
+  position: relative;
+  align-self: start;
+  height: 1.5rem;
+}
+
+.PrefixBadgePickerFill {
+  position: absolute;
+  left: -2px;
+  height: 100%;
+  border-radius: 1rem;
+  transition: background-color 0.18s ease-out;
+}
+
+.PrefixBadge {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 1.25rem;
+  height: 1.25rem;
+  flex: 0 0 1.25rem;
+  outline: none;
+}
+
+/*
+  TODO: look into why focus-visible shows when programmatically moving focus
+  https://github.com/w3c/csswg-drafts/issues/5885
+*/
+/* .PrefixBadge:focus-visible {
+  box-shadow: 0 0 0 0.125rem rgba(0, 123, 255, 0.25);
+} */
+
+.DefaultBadge {
+  background-color: var(--light-grey);
+  border-radius: 0.3rem;
+}
+
+.UnicornBadge {
+  width: 1rem;
+  height: 1rem;
+  background-image: url(~devtools/client/debugger/images/sources/unicorn-pixel.svg) !important;
+  background-size: auto 100%;
+}
+
+.ColorBadge {
+  display: grid;
+  place-items: center;
+  position: relative;
+  width: 0.8rem;
+  height: 0.8rem;
+  border-radius: 100%;
+  background-color: transparent;
+}
+
+.active.PrefixBadge:hover .ColorBadge::before,
+.active .ColorBadge::before,
+.ColorBadge::after {
+  grid-area: 1 / 1 / 1 / 1;
+  content: "";
+  display: block;
+  border-radius: 100%;
+  background-color: var(--primary-color);
+}
+
+.ColorBadge::before {
+  width: 100%;
+  height: 100%;
+  opacity: 0.2;
+}
+
+.ColorBadge::after {
+  width: 50%;
+  height: 50%;
+}
+
+/* Background fill colors */
```

Comment:
I tried moving most of this to CSS variables, but was having trouble getting things to work properly. I'm going to start reworking our theming soon so I'll be back in here to clean things up.